web-client/src/theme/variables.scss

```diff
@@ -8,69 +8,68 @@
 
 /** Ionic CSS Variables **/
 :root {
-  /** primary **/
-  --ion-color-primary: #3880ff;
-  --ion-color-primary-rgb: 56, 128, 255;
+  --ion-color-primary: #003450;
```

Comment:
I think it would be good to comment these colours with which UI Kit role / colour they correspond to, for easier maintenance.

I made a changeset for that while checking the colours: @jongbonga, can I add that for review?

Reply:
Yes